src/cert_context.rs

```diff
@@ -23,6 +24,20 @@ pub struct CertContext(winapi::PCCERT_CONTEXT);
 unsafe impl Sync for CertContext {}
 unsafe impl Send for CertContext {}
 
+// TODO: unsure if this is a solid decision instead of a fn `matches`
+// should work anyways
+impl cmp::PartialEq for CertContext {
+    fn eq(&self, other: &CertContext) -> bool {
+        let ret = unsafe {
+            crypt32::CertComparePublicKeyInfo(
```

Comment:
@bascule what's the most reasonable way to compare two certificates for equality? Looks like OSCrypto is comparing cert hashes: https://github.com/wbond/oscrypto/blob/master/oscrypto/_win/tls.py#L608